nmmo/render/render_client.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+import numpy as np
+
+from nmmo.render.websocket import Application
+from nmmo.render.overlay import OverlayRegistry
+from nmmo.render.render_utils import patch_packet
+
+
+# Render is external to the game
+class WebsocketRenderer:
+  def __init__(self):
+    # CHECK ME: It seems the renderer works fine without realm
+    self._client = Application(realm=None) # Do we need to pass realm?
+    self.overlay_pos = [256, 256]
+
+  def render(self, packet):
+    packet = {
+      'pos': self.overlay_pos,
+      'wilderness': 0,
+      **packet }
+
+    self.overlay_pos, _ = self._client.update(packet)
+
+
+class OnlineRenderer(WebsocketRenderer):
```

Comment:
I'm not sure what we get by subclassing here. Do we expect other types of Renders using Websocket?

I could imagine doing it the other way, where OnlineRenderer is the base class. If we do have other ways to render, we would probably want to keep the functionality in OnlineRenderer and change the output method. How about we combine them for now into once class?

Reply:
I combined into one class, named `WebsocketRenderer` to be explicit about websocket.

Two render functions were defined: `render_realm()` for online rendering, and `render_packet()` for replay rendering. I guess it would work for now.